drivers/clock_control/stm32_ll_clock.c

```diff
@@ -56,7 +56,8 @@ static inline int stm32_clock_control_on(struct device *dev,
 	case STM32_CLOCK_BUS_AHB1:
 		LL_AHB1_GRP1_EnableClock(pclken->enr);
 		break;
-#if defined(CONFIG_SOC_SERIES_STM32L4X) || defined(CONFIG_SOC_SERIES_STM32F4X)
+#if defined(CONFIG_SOC_SERIES_STM32L4X) || defined(CONFIG_SOC_SERIES_STM32F4X) \\
```

Comment:
Use '\' instead of '\\'.
This is making the CI failing

Reply:
yes '\\' is my mistake.

this pr was still working and can't build. I will fix those later. But I don't have a wellknow board so I haven't add a board.

Reply:
yes it is.

Reply:
Well, this is an issue for upstream, since we want to be able to build and test upstream code. Is there a way you could get a well know board?

Reply:
I think I can buy one.